src/TreeView/TreeView.tsx

```diff
@@ -408,6 +419,9 @@ const Item = React.forwardRef<HTMLElement, TreeViewItemProps>(
             style={{
               // @ts-ignore CSS custom property
               '--level': level,
+              contentVisibility: containIntrinsicSize ? 'auto' : undefined,
+              // @ts-ignore CSS custom property
+              'contain-intrinsic-size': containIntrinsicSize,
```

Comment:
```suggestion
              containIntrinsicSize,
```

Would it make sense for this to be `containIntrinsicSize` since we're setting an inline style? Otherwise I think we get the following error:

![image](https://user-images.githubusercontent.com/3901764/206003049-7724c5d4-f6d9-42a3-8ab6-71c507974170.png)

Reply:
updated!